nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
-            .displayName("SMTP hostname")
-            .description("The hostname to be embedded into the banner displayed when an " +
-                    "SMTP client connects to the processor TCP port .")
-            .required(true)
-            .expressionLanguageSupported(false)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    protected static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
             .name("SMTP_MAXIMUM_CONNECTIONS")
             .displayName("Maximum number of SMTP connection")
             .description("The maximum number of simultaneous SMTP connections.")
             .required(true)
+            .defaultValue("1")
             .expressionLanguageSupported(false)
             .addValidator(StandardValidators.INTEGER_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
```

Comment:
It will cause clients to hang around indefinitely 

Subethasmtp is a weird API. I don't truly understand the reason why it was coded like that but it transfers good chunks of the smtp conversation to the developer making use of it.